### ISSUE TITLE
Unfortunately doesn't work for me

### ISSUE THREAD

**Issue by Autark300** [peterdmv/anki-addons#1]
Hello,

I got very excited when I saw this add-on for the first time. But sadly it doesn't function for me. I type the ` exactly as described, but it doesn't convert the letters (a, e, i, o, u). My Anki version is up to date and the add on got installed without any issues too.  

Is this an issue on my end or does it generally not work right now?

All the best,

Tom

**Comment by peterdmv** [comment]
Hi, did you press the tab to switch the input field? The current version works for me with Anki 2.1.33.

**Comment by Autark300** [comment]
Hi,

Thank you very much for your answer! 

I'm on version 2.1.34 and I pressed tab as well, but it doesn't matter anymore, because it works perfectly now. I had to change the name of the card types to Latin. Seems like a pretty obvious mistake, but English is not my mother tongue so I initially typed "Latin" in my language and that - of course - didn't trigger the add-on. 

Thank you very much for coding this add-on!!
